src/tooling/piral-cli/src/injectors/pilet.ts

```diff
@@ -136,7 +140,7 @@ export default class PiletInjector implements KrasInjector {
     const indexHtml = readFileSync(target, 'utf8');
     
     // mechanism to inject server side debug piletApi config into piral emulator
-    const windowInjectionScript = `window['dbg:pilet-api'] = '${this.piletApi}';`;
+    const windowInjectionScript = `window['dbg:pilet-api'] = '${this.protocol}://localhost:${this.port}/$pilet-api';`;
```

Comment:
`/$pilet-api` should be flexible; just use `${this.config.api}` (I know confusing, but `this.config` was set to the provided *options*, while `config` in the constructor is the full kras configuration. We could also rename `this.config` to `this.options` to avoid the confusion.

Reply:
wow... can't believe I missed that...

Reply:
updated!